app/views/play/level/tome/SpellView.coffee

```diff
@@ -648,6 +654,8 @@ module.exports = class SpellView extends CocoView
     @spellThang = @spell.thangs[@thang.id]
     @createDebugView() unless @debugView
     @debugView?.thang = @thang
+    @createTranslationView() unless @translationView
+    @translationView?.thang = @thang
```

Comment:
It seems like we currently don't use `@thang` inside the translation view. I think we probably can avoid doing it in the future, too. So this (and line 25 of SpellTranslationView) can go. I guess it also doesn't use `@spell`, either, so that can also be removed.